### PR TITLE
str_num_copy -> str_truncate, as in Vanilla 0.7

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

**Comment by heinrich5991** [comment]
Called `str_truncate` in Teeworlds:

https://github.com/teeworlds/teeworlds/blob/e53b0cbb58a75da0df78b0ea4b902c854a4264ed/src/base/system.h#L791-L806

**Comment by def-** [comment]
@Learath2 Ok if we rename it to str_truncate?

**Comment by Learath2** [comment]
If we call it the same thing as tw, better make sure they behave the same and take their implementation. 

I have a feeling they don't behave the same from a quick skim over the code.

**Comment by Learath2** [comment]
@def- my bad, they look the same, we can just take that.

File: src/base/system.h [hunk 1]
```diff
@@ -997,20 +997,21 @@ void str_append(char *dst, const char *src, int dst_size);
...
-		Copies first num characters of a string to another.
+	Function: str_truncate
+		Truncates a string to a given length.
 
 	Parameters:
 		dst - Pointer to a buffer that shall receive the string.
-		src - String to be copied.
-		num - Number of characters to copy.
 		dst_size - Size of the buffer dst.
+		strc - String to be truncated.
```
Comment (heinrich5991) [inline src/base/system.h]:
```suggestion
		src - String to be truncated.
```

Apparently forgot to press the comment button.

**Pull request merged**